packages/xstate-vue/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@xstate/vue",
+  "version": "0.0.1",
+  "description": "XState tools for Vue",
+  "keywords": [
+    "state",
+    "machine",
+    "statechart",
+    "scxml",
+    "state",
+    "graph",
+    "vue",
+    "composable"
+  ],
+  "author": "David Khourshid <davidkpiano@gmail.com>",
+  "homepage": "https://github.com/davidkpiano/xstate/tree/master/packages/xstate-vue#readme",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "sideEffects": false,
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "files": [
+    "lib/**/*.js",
+    "lib/**/*.d.ts"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/davidkpiano/xstate.git"
+  },
+  "scripts": {
+    "clean": "rm -rf dist lib tsconfig.tsbuildinfo",
+    "prebuild": "npm run clean",
```

Comment:
Could you remove this one? It got removed in other packages - you have probably forked before it has happened.